js/analyzer.js

```diff
@@ -12,28 +12,63 @@ Analyzer = function (args){
  * Analyzer initialisation. Loads defaults and overloads custom settings.
  */
 Analyzer.prototype.init = function() {
+    this.initRequiredObjects();
+    this.initQueue();
+    this.loadWordlists();
+    this.setDefaults();
+    this.__output = [];
+};
+
+/**
+ * initializes required objects.
+ */
+Analyzer.prototype.initRequiredObjects = function(){
     //init preprocessor if not exists
-    if (typeof yst_preProcessor !== 'object' || yst_preProcessor.inputText !== this.config.textString) {
+    if (typeof yst_preProcessor !== "object" || yst_preProcessor.inputText !== this.config.textString) {
         yst_preProcessor = new PreProcessor(this.config.textString);
     }
+    //init helper
```

Comment:
removed this comment completely, since it is already in the function description.